client/me/concierge/shared/appointment-info.js

```diff
@@ -29,21 +38,68 @@ class AppointmentInfo extends Component {
 				'e.g. Thursday, December 20, 2018 at 8PM.',
 		} );
 
+		const conferenceLink = meta.conference_link || '';
+
 		return (
-			<Confirmation
-				title={ translate( 'Your upcoming appointment' ) }
-				description={ translate(
-					"You have a session with us on %(beginTime)s. It'll last about %(duration)d minutes, " +
-						'and we can talk about anything related to your site. Get all your questions ready ' +
-						'-- we look forward to chatting!',
-					{
-						args: {
-							beginTime: moment( beginTimestamp ).format( beginTimeFormat ),
-							duration: moment( endTimestamp ).diff( beginTimestamp, 'minutes' ),
-						},
-					}
-				) }
-			/>
+			<div>
+				<Confirmation
+					title={ translate( 'Your upcoming appointment' ) }
+					description={ translate(
+						'We can talk about anything related to your site. ' +
+							'Get all your questions ready ' +
+							'-- we look forward to chatting!',
+						{
+							args: {
+								beginTime: moment( beginTimestamp ).format( beginTimeFormat ),
+								duration: moment( endTimestamp ).diff( beginTimestamp, 'minutes' ),
+							},
+						}
+					) }
+				/>
+
+				<CompactCard className="shared__site-block">
+					<Site siteId={ site.ID } />
+				</CompactCard>
+				<CompactCard>
+					<FormattedHeader
+						subHeaderText={ translate( 'Your scheduled Business Concierge session details are:' ) }
+					/>
+
+					<FormFieldset>
+						<FormLabel>
+							{ translate(
+								'A link to start the session will appear here a few minutes before the session'
+							) }
+						</FormLabel>
+						<div className="shared__appointment-info-start-session">
+							<FormTextInput name="conferenceLink" value={ conferenceLink } disabled="disabled" />
+							{ conferenceLink && (
+								<a href={ conferenceLink } target="_blank" rel="noopener noreferrer">
+									<FormButton isPrimary={ true } type="button">
+										{ translate( 'Start session' ) }
+									</FormButton>
+								</a>
+							) }
+						</div>
+					</FormFieldset>
+
+					<FormFieldset>
+						<FormLabel>{ translate( 'When?' ) }</FormLabel>
```

Comment:
Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'When' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).